src/components/Accordion/Accordion.module.scss

```diff
@@ -109,36 +36,87 @@
   .Header {
     display: flex;
     width: 100%;
-    border-bottom: var(--zui-accordion-border-bottom);
+    border-bottom: 0.063rem solid #{$purple-50};
 
     &:hover {
       cursor: pointer;
+      border-bottom: 0.063rem solid #{$purple-50};
```

Comment:
nit: `#{}` shouldn't be required - `$purple-50`. Looks like these were just moved around though rather than new additions so not to worry.